app/src/main/java/org/simple/clinic/scanid/PatientPrefillInfoPayload.kt

```diff
@@ -0,0 +1,32 @@
+package org.simple.clinic.scanid
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class PatientPrefillInfoPayload(
```

Comment:
Also, correct me if I am wrong, we seem to be doing this in reverse order? We use the payload object for parsing then map it to `PatientPrefillInfo`. Currently, we are doing it another way around.

Reply:
Oh yaa,right. thanks for bringing this up.